core/bessctl.cc

```diff
@@ -1515,7 +1520,7 @@ class BESSControlImpl final : public BESSControl::Service {
       if ((wid != -1 && !is_worker_active(wid)) ||
           (wid == -1 && num_workers == 0)) {
         if (num_workers == 0 && (wid == 0 || wid == -1)) {
-          launch_worker(0, FLAGS_c);
+          launch_worker(0, FLAGS_c, "fast");
```

Comment:
Perhaps we can make "fast" as a default value for the third function parameter, and not specify it here..? At the moment all callers of `launch_worker()` must be updated whenever a default scheduler is switched.